lib/ExtensionManagerExtension.php

```diff
@@ -267,7 +267,9 @@ private function createComposer(Container $container): Composer
     private function registerService(ContainerBuilder $container)
     {
         $container->register('extension_manager.service.progress', function (Container $container) {
-            $output = $this->isRpcCommand($container) ? new BufferedOutput() : $container->get(ConsoleExtension::SERVICE_OUTPUT);
+            $output = $this->isRpcCommand($container) || $container->getParameter(self::PARAM_QUIET)
+                ? new BufferedOutput()
+                : $container->get(ConsoleExtension::SERVICE_OUTPUT);
```

Comment:
tbh we should probably be doing something Symfony speicfic here to surpress output (i.e. using the applications `-q` option)